src/Tree/HydroTree.cpp

```diff
@@ -230,7 +230,6 @@ void HydroTree<ndim,ParticleType>::AllocateMemory
     levelneibbuf    = new int*[Nthreads];
     activepartbuf   = new ParticleType<ndim>*[Nthreads];
     neibpartbuf     = new ParticleType<ndim>*[Nthreads];
```

Comment:
I deleted neibpartbuf in the other commit but you've got rid of cellbuf but left neibpartbuf.  Do we use neibpartbuf anywhere (or any plans to keep it to use it later)?  Otherwise I'll delete it again in the merge.